app/mailers/approval_notifier_mailer.rb

```diff
@@ -23,18 +23,37 @@ def reset_password_approval_email(user_id:)
 
   private
 
+  # permissions_users are admins as per the old permissions system
+  # accesses_users are admins as per the new permissions system
+  # we need to send emails to the superset of the two till we flip everyone over to new permissions
+
   def supervisor_emails
-    users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.facility_group).map(&:user)
+    permissions_users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.facility_group).map(&:user)
+    accesses_users = User.admins.where(access_level: :manager)
+      .select { |admin| admin.accessible_facilities(:manage).include?(user.facility) }
+      .reject { |admin| admin.accesses.map(&:resource_type).include?("Organization") }
+
+    users = (permissions_users + accesses_users).uniq.compact
     users.map(&:email).join(",")
   end
 
   def organization_owner_emails
-    users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.organization).map(&:user)
+    permissions_users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.organization).map(&:user)
+    accesses_users = User.admins.where(access_level: :manager)
+      .select { |admin|
+      admin.accessible_facilities(:manage).include?(user.facility) &&
+        admin.accesses.map(&:resource_type).include?("Organization")
```

Comment:
I feel like for this check you really want to ensure the admin manages the Organization that the facility is in, and not just any Org.